src/meta/src/barrier/command.rs

```diff
@@ -663,7 +670,35 @@ impl CommandContext {
             Command::CancelStreamingJob(table_fragments) => {
                 let node_actors = table_fragments.worker_actor_ids();
                 self.clean_up(node_actors).await?;
-                // Drop fragment info in meta store.
+
+                let table_id = table_fragments.table_id().table_id;
```

Comment:
May I ask why don't have this procedure before? May add some comments for this.

Reply:
Added